modules/phase_field/test/tests/GrandPotentialPFM/GrandPotentialStrictMassConservation.i

```diff
@@ -0,0 +1,230 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 34
+  ny = 34
+  xmin = 0
+  xmax = 340
+  ymin = 0
+  ymax = 340
+[]
+
+[GlobalParams]
+  op_num = 2
+  var_name_base = gr
+  int_width = 20
+[]
+
+[Variables]
+  [./w]
+  [../]
+  [./c]
+  [../]
+  [./phi]
+  [../]
+  [./PolycrystalVariables]
+  [../]
+[]
+
+[AuxVariables]
+  [./bnds]
+  [../]
+  [./T]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[ICs]
+  [./phi_IC]
+    type = SpecifiedSmoothCircleIC
+    variable = phi
+    x_positions = '170 170'
+    y_positions = ' 70 270'
+    z_positions = '  0   0'
+    radii = '100 100'
+    invalue = 0
+    outvalue = 1
+  [../]
+  [./c_IC]
+    type = SpecifiedSmoothCircleIC
+    variable = c
+    x_positions = '170 170'
+    y_positions = ' 70 270'
+    z_positions = '  0   0'
+    radii = '100 100'
+    invalue = 0
+    outvalue = 1
+  [../]
+  [./gr0_IC]
+    type = SmoothCircleIC
+    variable = gr0
+    x1 = 170
+    y1 = 70
+    z1 = 0
+    radius = 100
+    invalue = 1
+    outvalue = 0
+  [../]
+  [./gr1_IC]
+    type = SmoothCircleIC
+    variable = gr1
+    x1 = 170
+    y1 = 270
+    z1 = 0
+    radius = 100
+    invalue = 1
+    outvalue = 0
+  [../]
+[]
+
+[Functions]
```

Comment:
you can set T = 1600 in the initial conditions for the AuxVariable so you can remove this to keep the input file short